release-engineering-analytics.Rmd

```diff
@@ -80,10 +97,44 @@ _Table 1. Papers found indirectly by investigating citations of/by other papers.
 | @khomh2015a    | is cited by | @poo-caama2016a <br> @teixeira2017a |
 | @mantyla2015a  | is cited by | @rodriguez2017a <br> @cesar2017a    |
 
+### Study Selection
+
 In order to aggregate our collective efforts, selected sources were stored in a
 custom built web-based tool for conducting literature reviews.
 The source code of this tool is published in a
 [GitHub repository](https://github.com/jessetilro/research).
 By commenting on and tagging our findings we were able to export a filtered
 list of publications,
 the relevance of which was agreed upon.
+
+### Study Quality Assessent
+
+<!-- TODO -->
+
+### Data Extraction
+
+<!-- TODO -->
+
+### Data Synthesis
+
+<!-- TODO -->
+
```

Comment:
Do you think it would make sense to add a section about _Included and Excluded Studies_ here? (Kitchenham Table 9, structure of reports of systematic reviews.)

I think it might be nice to split listings of retrieved literature from the search strategy, but I'm not sure.

Reply:
Yes, I agree.